music21/environment.py

```diff
@@ -1268,8 +1268,8 @@ def __setitem__(self, key, value):
         if key in self._environment.getKeysToPaths():
             # try to expand user if found; otherwise return unaltered
             if value is not None and not str(value).startswith('/skip'):
-                value = common.cleanpath(value, returnPathlib=False)
-                if not os.path.exists(value):
+                value = common.cleanpath(value, returnPathlib=True)
+                if not value.exists():
```

Comment:
Does `self._envrionment.__setitem__` work with a Pathlib object as a key?  Not sure.

Reply:
Yes, but it programmatically converts to string and then when retrieving programmatically converts back to pathlib. So there's no point to this change, I'll revert it. Thanks.